### ISSUE TITLE
Try / finally formatted on a single line

### ISSUE THREAD

**Issue by nrinaudo** [scalameta/scalafmt#1002]
- **Version**: 1.1.0
- **Integration**: command line (coursier)
- **Configuration**: default (no configuration specified)

## Steps
Input
```scala
object Test {
  try { foo() }
  finally { bar() }
}
```

## Problem

Scalafmt formats code like this:
```scala
object Test {
  try { foo() } finally { bar() }
}
```

## Expectation

I was expecting `finally` to have a line of its own.

## Notes

This might be, if not a duplicate of, heavily related to #350.

**Comment by olafurpg** [comment]
I suspect this is even related to https://github.com/scalameta/scalafmt/issues/271

I once tried to take a stab at it but scalafmt's design/implementation makes it surprisingly hard to insert a newline there.

**Comment by olafurpg** [comment]
Fixed in #1187 🎉 

Scalafmt will now force try/finally blocks to break on multiple lines

```scala
try { foo() } finally { bar() }
// after
try {
  foo()
} finally {
  bar()
}

```

This is not an optimal solution but avoids the ugly output at least.
